package.json

```diff
@@ -101,6 +101,7 @@
     "lodash-es": "^4.17.21",
     "npm-run-all": "^4.1.5",
     "prettier": "^3.2.5",
+    "puppeteer-core": "^22.15.0",
```

Comment:
The latest is 24.x, I set it to 22.x to be in sync with what webdriverio wants: https://github.com/webdriverio/webdriverio/blob/8908cc4e3cd75e85bf47692477fc5c0ab58f9168/packages/webdriverio/package.json#L105